packages/create-waku/src/index.ts

```diff
@@ -4,21 +4,45 @@ import path from 'node:path';
 import { fileURLToPath } from 'node:url';
 import { parseArgs } from 'node:util';
 import { default as prompts } from 'prompts';
-import { red, green, bold } from 'kolorist';
+import { red, green, bold, cyan } from 'kolorist';
 import fse from 'fs-extra/esm';
 import checkForUpdate from 'update-check';
 import { createRequire } from 'node:module';
+import {
+  downloadAndExtractExample,
+  downloadAndExtractRepo,
+  existsInRepo,
+  getRepoInfo,
+  hasRepo,
+  type RepoInfo,
+} from './helpers/examples';
 
 // FIXME is there a better way with prompts?
-const { values } = parseArgs({
+const { tokens } = parseArgs({
   args: process.argv.slice(2),
   options: {
-    'choose-template': {
-      type: 'boolean',
+    example: {
+      type: 'string',
     },
   },
+  tokens: true,
 });
 
+function isErrorLike(err: unknown): err is { message: string } {
+  return (
+    typeof err === 'object' &&
+    err !== null &&
+    typeof (err as { message?: unknown }).message === 'string'
+  );
+}
+// FIXME no-nay
+function getExample(tokens: any) {
+  const exampleToken = tokens.find(
+    (token: any) => token.kind === 'option' && token.name === 'example',
+  );
+  if (exampleToken) return exampleToken.value;
+  else return;
```

Comment:
We prefer curly blocks.
```suggestion
  if (exampleToken) {
    return exampleToken.value;
  }
  return;
```

Reply:
I prefer this:
```suggestion
  return exampleToken?.value;
```